hypha/apply/users/decorators.py

```diff
@@ -27,6 +39,10 @@ def is_approver(user):
 
 staff_required = [login_required, user_passes_test(is_apply_staff)]
 
+finance_required = [login_required, user_passes_test(is_finance)]
+
+staff_or_finace_required = [login_required, user_passes_test(is_apply_staff_or_finance)]
```

Comment:
Typo for finance in `staff_or_finace_required`

Reply:
Thanks! Fixed now I believe.